tools/tpm2_nvread.c

```diff
@@ -158,8 +158,8 @@ static bool nv_read(TSS2_SYS_CONTEXT *sapi_context) {
         ctx.size_to_read -= nv_data.t.size;
         ctx.offset += nv_data.t.size;
 
-        memcpy(data_buffer, nv_data.t.buffer, data_offest);
-        data_offest += nv_data.t.size;
+        memcpy(data_buffer + data_offset, nv_data.t.buffer, nv_data.t.size);
+        data_offset += nv_data.t.size;
     }
```

Comment:
@cardoe I also noticed this issue but didn't dig further. Thanks for the fix, it looks good to me.